app/src/main/java/org/oppia/android/app/options/AudioLanguageActivityPresenter.kt

```diff
@@ -13,6 +14,14 @@ class AudioLanguageActivityPresenter @Inject constructor(private val activity: A
 
   fun handleOnCreate(prefKey: String, prefValue: String) {
     activity.setContentView(R.layout.audio_language_activity)
+    val toolbar: androidx.appcompat.widget.Toolbar = activity.findViewById(R.id.audio_language_toolbar)
```

Comment:
Please use databinding here and also convert the activity to use DataBinding.

Also do not use `androidx.appcompat.widget.Toolbar` instead use `Toolbar` and import it at top.

Reply:
Hey @rt4914, I completed the above changes and pushed the code.